### PR TITLE
Remove temporary files after processing

### PR DESCRIPTION
**Pull request by soulchild**
Fixes #5450

### CODE REVIEW

File: server/queues/tasks/ExportTask.ts [hunk 1]
```diff
@@ -93,6 +95,12 @@ export default abstract class ExportTask extends BaseTask<Props> {
         }).schedule();
       }
       throw error;
+    } finally {
+      if (filePath) {
+        await fs.promises.unlink(filePath).catch((error) => {
+          Logger.error("Failed to delete temporary file ${filePath}", error);
```
Comment (tommoor) [inline server/queues/tasks/ExportTask.ts:101]:
```suggestion
          Logger.error(`Failed to delete temporary file ${filePath}`, error);
```

File: server/queues/tasks/ExportTask.ts [hunk 2]
```diff
@@ -93,6 +95,12 @@ export default abstract class ExportTask extends BaseTask<Props> {
         }).schedule();
       }
       throw error;
+    } finally {
+      if (filePath) {
+        await fs.promises.unlink(filePath).catch((error) => {
```
Comment (tommoor) [inline server/queues/tasks/ExportTask.ts:100]:
```suggestion
        void fs.promises.unlink(filePath).catch((error) => {
```

Probably no benefit to waiting here

**Comment by soulchild** [comment]
@tommoor Sorry for taking so long to fix the typing issue. Is there something I need to do to make the `license/cla` step pass?

**Review by tommoor — APPROVED** [review]

**Comment by tommoor** [comment]
@soulchild usually just sign the CLA, not sure why it didn't pick up – perhaps because it stops checking after a few days

**Pull request merged**